algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -228,6 +228,97 @@ class CMSBigIntTest extends CMSTest[BigInt]
 class CMSStringTest extends CMSTest[String]
 class CMSBytesTest extends CMSTest[Bytes]
 
+abstract class CmsProperty[K] extends ApproximateProperty
+
+object CmsProperty {
+  val delta = 1E-10
+  val eps = 0.001
+  val seed = 1
+
+  def makeApproximate[K: CMSHasher](exact: Vector[K]) = {
+    val cmsMonoid: CMSMonoid[K] = CMS.monoid(eps, delta, seed)
+    cmsMonoid.sum(exact.map(cmsMonoid.create(_)))
+  }
+}
+
+abstract class CmsFrequencyProperty[K: CMSHasher: Gen] extends CmsProperty {
+  type Exact = Vector[K]
+  type Approx = CMS[K]
+
+  type Input = K
+  type Result = Long
+
+  def makeApproximate(e: Exact) = CmsProperty.makeApproximate(e)
+  def inputGenerator(e: Vector[K]): Gen[K] = Gen.oneOf(e)
+
+  def exactResult(list: Vector[K], key: K) = list.filter(_ == key).length
```

Comment:
can we call `list` `vec` instead? Also, `vec.count(_ == key)` is going to be faster here since it does not materialize a second vector.